tiledb/sm/filesystem/azure.cc

```diff
@@ -158,18 +158,18 @@ Status Azure::init(const Config& config, ThreadPool* const thread_pool) {
   // 'curl_easy_init'. This ensures that our 'thread_pool_' threads
   // will not block on the blob client's internal request queue
   // unless the user is performing concurrent I/O on this instance.
-#ifdef __linux__
+
   // Get CA Cert bundle file from global state. This is initialized and cached
   // if detected. We have only had issues with finding the certificate path on
   // Linux.
-  const std::string cert_file =
-      global_state::GlobalState::GetGlobalState().cert_file();
-  client_ = make_shared<azure::storage_lite::blob_client>(
-      HERE(), account, thread_pool_->concurrency_level(), cert_file);
-#else
-  client_ = make_shared<azure::storage_lite::blob_client>(
+  if constexpr (tiledb::platform::PlatformCertFile::enabled) {
```

Comment:
I know you just changed it from `exists` to `enabled`, but `maybe_available` is even more descriptive.

Reply:
How strongly do you feel on this one? I don't mind the change but I personally prefer enabled as good enough and also shorter.

Reply:
Don't care a lot.

Reply:
Leaving as is then.